### PR TITLE
list all active jobs

### PR DESCRIPTION
**Pull request by apelstein**

### CODE REVIEW

File: src/tons_cli/job_group.py [hunk 1]
```diff
@@ -17,8 +17,21 @@ def list_jobs():
...
+def list_jobs():
+    """List all active jobs"""
+    jobs = sdk().get_all_active_jobs()
+    if jobs is None:
+        click.echo("Failed to get active jobs", err=True)
+    else:
+        if len(jobs) == 0:
+            click.echo(f"No active jobs found")
+        for job_to_echo in jobs:
+            click.echo(job_to_echo)
```
Comment (daTokenizer) [inline src/tons_cli/job_group.py:34]:
```suggestion
        for job in jobs:
            click.echo(job)
```

Comment (daTokenizer) [reply src/tons_cli/job_group.py:34]: not sure you want to dump everything here to the user, filter out only what you see in the jobs screen (+latest APM)

**Review by daTokenizer — CHANGES_REQUESTED** [review]

File: src/tons_cli/job_group.py (hunk 1 above)
Comment (apelstein) [reply src/tons_cli/job_group.py:34]: image
 is this good?

Comment (apelstein) [reply src/tons_cli/job_group.py:34]: with APM:
image

File: src/tons_cli/job_group.py [hunk 2]
```diff
@@ -21,6 +21,19 @@ def list_jobs():
...
+    """List all active jobs"""
+    jobs = sdk().get_all_active_jobs()
+    if jobs is None:
+        click.echo("Failed to get active jobs", err=True)
+    else:
+        if len(jobs) == 0:
+            click.echo(f"No active jobs found")
+        for job in jobs:
+            click.echo(f"Job: {job.job_name} Status: [{job.status}] Start time: {job.start_time}")
+            click.echo(f"Latest APM: {job.latest_apm}")
```
Comment (daTokenizer) [inline src/tons_cli/job_group.py:35]: use [prettytable](https://ptable.readthedocs.io/en/latest/tutorial.html) here instead of printing line by line

**Review by daTokenizer — CHANGES_REQUESTED** [review]

**Review by daTokenizer — APPROVED** [review]

**Pull request merged**